curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-54-poker-hands.english.md

```diff
@@ -22,12 +22,15 @@ The cards are valued in the order:2, 3, 4, 5, 6, 7, 8, 9, 10, Jack, Queen, King,
 If two players have the same ranked hands then the rank made up of the highest value wins; for example, a pair of eights beats a pair of fives (see example 1 below). But if two ranks tie, for example, both players have a pair of queens, then highest cards in each hand are compared (see example 4 below); if the highest cards tie then the next highest cards are compared, and so on.
 Consider the following five hands dealt to two players:
 
-Hand Player 1 Player 2 Winner
-1 5H 5C 6S 7S KDPair of Fives 2C 3S 8S 8D TDPair of Eights Player 2
-2 5D 8C 9S JS ACHighest card Ace 2C 5C 7D 8S QHHighest card Queen Player 1
-3 2D 9C AS AH ACThree Aces 3D 6D 7D TD QDFlush  with Diamonds Player 2
-4 4D 6S 9H QH QCPair of QueensHighest card Nine 3D 6D 7H QD QSPair of QueensHighest card Seven Player 1
-5 2H 2D 4C 4D 4SFull HouseWith Three Fours 3C 3D 3S 9S 9DFull Housewith Three Threes Player 1
+|Hand||Player 1||Player 2||Winner|
+|--- |--- |--- |--- |--- |--- |--- |
+|<strong>1</strong>||5H 5C 6S 7S KD <br> Pair of Fives||2C 3S 8S 8D TD <br> Pair of Eights||Player 2|
+|<strong>2</strong>||5D 8C 9S JS AC <br> Highest card Ace||2C 5C 7D 8S QH <br> Highest card Queen||Player 1|
+|<strong>3</strong>||2D 9C AS AH AC <br> Three Aces||3D 6D 7D TD QD <br> Flush with Diamonds||Player 2|
+|<strong>4</strong>||4D 6S 9H QH QC <br> Pair of Queens <br> Highest card Nine||3D 6D 7H QD QS <br> Pair of Queens <br> Highest card Seven||Player 1|
+|<strong>5</strong>||2H 2D 4C 4D 4S <br> Full House <br> with Three Fours||3C 3D 3S 9S 9D <br> Full House <br> with Three Threes||Player 1|
+
+The `handsArr` array contains one-thousand random hands dealt to two players. Each string contains ten cards: the first five are Player 1's cards and the last five are Player 2's cards. You can assume that all hands are valid (no invalid characters or repeated cards), each player's hand is in no specific order, and in each hand there is a clear winner.
 
 The file, poker.txt, contains one-thousand random hands dealt to two players. Each line of the file contains ten cards (separated by a single space): the first five are Player 1's cards and the last five are Player 2's cards. You can assume that all hands are valid (no invalid characters or repeated cards), each player's hand is in no specific order, and in each hand there is a clear winner.
```

Comment:
This should be handled over in #38016. There the text reads:

```
The `handsArr` array contains one-thousand random hands dealt to two players. Each string contains ten cards: the first five are Player 1's cards and the last five are Player 2's cards. You can assume that all hands are valid (no invalid characters or repeated cards), each player's hand is in no specific order, and in each hand there is a clear winner.
```

Speaking of which, I wasn't sure if it was better to include all of these changes over in that PR or spin it off into a different one like I did here. Let me know if you think it's better to roll all of these changes into that other big PR.

Reply:
Broadly speaking, I find it works best if a PR is as focused as possible.  However, the atomic unit is really the commit, so as long as each commit does one thing, it's fine.

So, ideally, this would be two commits:

1. changes to the css to make generic tables look good
2. changes to tables themselves, html -> markdown

Not a big deal, of course, as they're obviously linked - that's just how I'd do it!

Regarding 1 vs 2 PRs, 2 seems fine.  Although, I'd restrict this one to "changes to tables", so ignore my original suggestion and get rid of line 33, since that's handled by #38016.  What do you reckon?

Reply:
I unresolved it so it's easier to find - Github's terrible when it comes to finding resolved conversations!

Reply:
Awesome, thank you. I'll keep that in mind for future PRs. It makes sense to keep things as focused as possible. And yes, things do get lost in the shuffle easily when conversations are resolved. Thank you for reopening this one!

I'll update this so it gets rid of line 33 like you suggest.